content/documentation/standards.md

```diff
@@ -0,0 +1,6 @@
+## Documentation Standards
```

Comment:
Did you mean to commit this file with this PR?

Reply:
@osterman  no i didn't. This was part of my commit issues from before.

Reply:
You can just delete it from your commit and push.